bin/instruments/qmi_tenma_72psu.py

```diff
@@ -0,0 +1,150 @@
+#! /usr/bin/env python3
```

Comment:
all our other CLI tools don't have .py appended to their filenames

Reply:
Good catch, I'll remove the extension!